SysKit.XCellKit/SpreadsheetStyle.cs

```diff
@@ -21,25 +20,23 @@ public string GetIdentifier()
             var sb = new StringBuilder();
             if (BackgroundColor.HasValue)
             {
-                var colorRgb = BackgroundColor.Value.ToArgb();
-                sb.Append(colorRgb);
+                sb.Append(BackgroundColor.Value.GetRgbAsInt());
             }
             if (ForegroundColor.HasValue)
             {
-                var colorRgb = ForegroundColor.Value.ToArgb();
-                sb.Append(colorRgb);
+                sb.Append(ForegroundColor.Value.GetRgbAsInt());
             }
             if (Font != null)
             {
                 var fontid = Font.ToString();
                 sb.Append(fontid);
-                sb.Append(Font.Style);
+                // sb.Append(Font.Style);
```

Comment:
what does this do? will we miss the style property?

Reply:
This code appends font style to style identifier. I fixed it so it works with SkiaSharp library.